langchain/utilities/bash.py

```diff
@@ -1,24 +1,59 @@
 """Wrapper around subprocess to run commands."""
+import re
 import subprocess
 from typing import List, Union
+from uuid import uuid4
+
+import pexpect
```

Comment:
Hi 👋 late comment here. Does `pexpect` need to be added to dependencies? I believe this is breaking `0.0.151` (related issue here: https://github.com/hwchase17/langchain/issues/3664)

Reply:
I'm also running into this.